_includes/install_instructions/python.html

```diff
@@ -0,0 +1,101 @@
+<div id="python"> {% comment %} Start of 'Python' section. Remove the third paragraph if
```

Comment:
I suggest moving this comment outside to the beginning of the file as you did in `git.html`